hs-bindgen/test/th/Test/TH/Test01.hs

```diff
@@ -20,6 +20,7 @@ import HsBindgen.TH
 
 -- Used by generated code
 import Foreign.C.Types
+import HsBindgen.Runtime.Prelude qualified
```

Comment:
I'm not sure why the `Prelude` is included, but then at least `SizedByteArray` is probably not needed. Nor `Foreign.C.Types`